src/Files.App/Data/Models/CompressArchiveModel.cs

```diff
@@ -161,7 +161,8 @@ public async Task<bool> RunCreationAsync()
 			//Use UTF-8 encoding. 
 			//References: 7-zip chm --> Command Line Version --> Switches
 			//--> -m --> cu=[off | on].
-			compressor.CustomParameters.Add("cu", "on");
+			if (FileFormat != ArchiveFormats.SevenZip)
```

Comment:
This looks good, can you please add a code comment linking to the issue for future reference?

Reply:
Thanks again!